src/components/sde/tests/Minimal/Minimal_Test++.cpp

```diff
@@ -6,28 +6,40 @@
 #include "sde_lib.h"
 #include "sde_lib.hpp"
 
-long long local_var;
 
-void mintest_init(void){
-    papi_sde::PapiSde sde("Min Example Code in C++");
+////////////////////////////////////////////////////////////////////////////////
+//------- Library example that exports SDEs
+
+class MinTest{
+    private:
```

Comment:
Based on the [ICL style guide](https://github.com/icl-utk-edu/style/blob/master/ICL_C_CPP_Coding_Style_Guide.md) both `private:` and `public:` should not be indented.